src/OpenTelemetry.Instrumentation.AspNetCore/AspNetCoreInstrumentationOptions.cs

```diff
@@ -53,5 +53,10 @@ public class AspNetCoreInstrumentationOptions
         /// The type of this object depends on the event, which is given by the above parameter.</para>
         /// </remarks>
         public Action<Activity, string, object> Enrich { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether the exception will be recorded or not.
+        /// </summary>
```

Comment:
will be recorded as `ActivityEvent` or not.
Add link to spec: https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/exceptions.md

Reply:
done!